src/translations/main_pl.ts

```diff
@@ -465,19 +465,19 @@ OpenSSL może nie być zainstalowane.</translation>
     </message>
     <message>
         <source>Start Thunderbird by clicking on the tray icon if it was closed</source>
-        <translation type="unfinished"></translation>
+        <translation>Uruchom Thunderbird klikając w ikonę na tecce systemowej jeśli został zamknięty.</translation>
```

Comment:
```suggestion
        <translation>Uruchom Thunderbird klikając w ikonę na tecce systemowej jeśli został zamknięty</translation>
```
In main the settings we generally don't add dots at the end.